.gitignore

```diff
@@ -153,4 +153,7 @@ xcuserdata
 !*.xcworkspace/contents.xcworkspacedata
 **/xcshareddata/WorkspaceSettings.xcsettings
 
-# End of https://www.toptal.com/developers/gitignore/api/macos,swift,xcode,carthage,cocoapods,swiftpm
\ No newline at end of file
+# End of https://www.toptal.com/developers/gitignore/api/macos,swift,xcode,carthage,cocoapods,swiftpm
+
+# SwiftLint Remote Config Cache
+.swiftlint/RemoteConfigCache
```

Comment:
I am not sure this should be under source control.

Reply:
It was auto added. cache should  be ignored.

Reply:
This is still here. Correct me if I'm wrong but gitignore is a manual maintenance file, so I'm not sure why this hasn't been removed if it was auto-added at somepoint but you agree cache should be ignored?

Reply:
It was auto added to `.gitignore` by `swiftlint`. `gitignore` can be changed by any script.